x-pack/plugin/rollup/src/internalClusterTest/java/org/elasticsearch/xpack/downsample/DownsampleTransportFailureIT.java

```diff
@@ -339,40 +354,4 @@ public void testDownsampleActionExceptionDisruption() {
         ensureStableCluster(testCluster.size());
         assertDownsampleFailure(testCluster.coordinatorName());
     }
-
-    public void testRollupIndexerActionExceptionDisruption() {
```

Comment:
Maybe also add a test with multiple nodes and performs a rolling restart while performing downsampling?
Doesn't have to be in this class, but can also be in a new test class.

Reply:
I wrote tests doing full and rolling restarts. Consider that, anyway, if the indexing shard (where we write downsample data) is not ready when we send the bulk request the task will incur into a failure.